jwt/api.py

```diff
@@ -0,0 +1,196 @@
+import binascii
+
+from calendar import timegm
+from collections import Mapping
+from datetime import datetime, timedelta
+
+from .utils import base64url_decode, base64url_encode
```

Comment:
Super minor, but I'd flip around the .utils and .compat imports so they're alphabetical.